src/autosuggest/interfaces.ts

```diff
@@ -122,6 +122,7 @@ export namespace AutosuggestProps {
   export type StatusType = DropdownStatusProps.StatusType;
   export interface SelectDetail {
     value: string;
+    selectedOption?: Option;
```

Comment:
[minor] would it be worthwhile doing a dry run to live to check if any consumers are misusing these interfaces, so we can notify them in advance?